README.md

```diff
@@ -4,5 +4,40 @@ A Python library to integrate automatic differentiation tools such as Jax with Q
 
 This package is a work in progress. Feel free to take part in the discussions by opening new issues.
 
+## Installation
+
+### PyPI version
+To install ``qgrad``
+
+```
+pip install qgrad
+```
+
+### From source
+To install ``qgrad`` development version, clone this repository and from the terminal type
+
+```
+python setup.py develop
+```
+
+### Requirements
+``qgrad`` dependencies are automatically installed with `pip`. They are:
+
+``numpy scipy matplotlib cython pytest qutip jax``
+
+
+## About
+
+``qgrad`` is a library that implements Hamiltonian learning in the context of quantum physics-based optimization tasks.
+ ``qgrad`` reproduces essential [QuTiP](http://qutip.org/) functions to reduce the friction for existing QuTiP users.
+
+While many quantum libraries focus on quantum circuit learning, ``qgrad`` focuses on Hamiltonian learning.
```

Comment:
@quantshah suggested removing this comparision. I have removed this from the home docs as well

Reply:
I should have been clearer, I was talking about line 34